sphinx/locale/Makefile

```diff
@@ -0,0 +1,11 @@
+IGNORED = ta
```

Comment:
why do we ignore this language?

Reply:
Because its errors are also ignored in the compilation script, and it currently indeed has a lot:

https://github.com/sphinx-doc/sphinx/blob/90574bc96693e6ef7aa9d2e028a84055a784d0e9/utils/babel_runner.py#L232

I don't know what would be the best solution, but currently I chose to ignore it as well.

Reply:
Oh, then could you maybe add a comment for the rationale + something like "see babel_runner" (don't we have an issue for this?)

Reply:
Done